src/App.jsx

```diff
@@ -12,6 +13,7 @@ function App() {
     <Route path='/' element={<Home />} />
     <Route path='/articles' element={<ArticlesList />} />
     <Route path='/topics' element={<Topics />} />
+    <Route path='articles/:article_id' element={<SingleArticle />} />
```

Comment:
The paths should be consistent - although there is no functional difference here, the absence of the first forward slash means that this is a relative path not an absolute path.